app/api/test/module/Api/src/Domain/CommandHandler/Bus/Ebsr/ProcessRequestMapTest.php

```diff
@@ -1,34 +1,24 @@
 <?php
 
-declare(strict_types=1);
-
```

Comment:
Don't need to remove this. Github wouldn't allow to post a suggested change for whatever reason.